server/controllers/user/login.php

```diff
@@ -140,7 +140,7 @@ private function clearOldRememberTokens() {
     private function createRememberToken() {
         $remember = Controller::request('remember');
 
-        if (!Controller::request('staff') && $remember) {
+        if($remember) {
```

Comment:
we need to add the field 'staff' to sessioncookie, you're currently treating it as user.

Also When logging in with `rememberToken ` it will try to retrieve an user based on `userId`